xla/backends/gpu/codegen/custom.cc

```diff
@@ -547,44 +540,50 @@ absl::StatusOr<FusionEmissionResult> EmitGemm(
   std::optional<HloInstruction*> while_op =
       GetParentWhileOp(fusion, call_graph);
   std::unique_ptr<HloModule> init_module, update_module;
+  std::optional<int64_t> indvar_idx = std::nullopt;
```

Comment:
nit: you don't need to do `=  std::nullopt`, this is equivalent to default initialization.